src/python/integration/deploy_python_app_with_dynatrace_test.go

```diff
@@ -26,7 +27,11 @@ var _ = Describe("CF Python Buildpack", func() {
 		}
 
 		dynatraceAPI = cutlass.New(Fixtures("fake_dynatrace_api"))
-
+		// TODO: remove this once go-buildpack runs on cflinuxfs4
```

Comment:
I'm confused by the reference to: `go-buildpack`.

Shouldn't this comment read:

```
// TODO: remove this once the dynatrace buildpack runs on cflinuxfs4
```

Reply:
I agree, it is a bit confusing. Basically what you mean is:

When Go's Buildpack has support for cflinuxfs4 (call it support for running on cflinuxfs4 and also available in the environment via cf buildpacks) this should be removed.

When I mean available, is that when you run `cf buildpacks` command in the environment, it will show 2 entries per buildpack, 1 for cflinuxfs and 1 for cflinuxfs3. Something like this:

```bash
❯ cf buildpacks.
Getting buildpacks as admin...

position name stack enabled locked filename
1 staticfile_buildpack cflinuxfs3 true false staticfile_buildpack-cflinuxfs3-v1.5.35.zip
2 staticfile_buildpack cflinuxfs4 true false staticfile_buildpack-cflinuxfs4-v1.5.35.zip
3 java_buildpack cflinuxfs3 true false false java-buildpack-cflinuxfs3-v4.53.zip
4 java_buildpack cflinuxfs4 true false false java-buildpack-cflinuxfs4-v4.53.zip
5 ruby_buildpack cflinuxfs3 true false false ruby_buildpack-cflinuxfs3-v1.8.60.zip
5 ruby_buildpack cflinuxfs4 true false false ruby_buildpack-cflinuxfs4-v1.8.60.zip
.
.
.
```